migrations/V83__Add_Accessibility_Notes.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE accessibility_notes (
+    id uuid PRIMARY KEY NOT NULL,
```

Comment:
```suggestion
    id uuid PRIMARY KEY,
```

(nit) primary keys in Postgres [don't allow for null values](https://www.postgresql.org/docs/12/ddl-constraints.html#DDL-CONSTRAINTS-PRIMARY-KEYS) and don't need to be explicitly NOT NULL